includes/payment-methods/class-upe-payment-gateway.php

```diff
@@ -924,7 +924,7 @@ function() use ( $payment_fields ) {
 	 */
 	public function get_payment_method_ids_enabled_at_checkout( $order_id = null, $force_currency_check = false ) {
 		$capture                    = empty( $this->get_option( 'manual_capture' ) ) || $this->get_option( 'manual_capture' ) === 'no';
```

Comment:
While we are touching on this, the variable name is misleading. Can you tweak the name, please?